visualdl/server/visualDL

```diff
@@ -298,11 +298,18 @@ def individual_audio():
 
 @app.route('/data/plugin/graphs/graph')
 def graph():
-    # TODO(ChunweiYan) need to add a config for whether have graph.
-    if graph_image_path is None or not os.path.isfile(graph_image_path):
-        data = {'url': ''}
+    # TODO(daming-lu): rename variables because we switched from static graph generated by graphviz
+    # to d3/dagre drawn svg
+    if graph_image_path is not None:
+        data = {
+            'data': graph_image_path,
+        }
     else:
-        data = {'url': '/graphs/image'}
+        image_dir = os.path.join(args.logdir, "graphs")
```

Comment:
image_dir seems un-necessary now.

Reply:
Yes. It will be removed in the code cleanup.